.travis.yml

```diff
@@ -6,8 +6,11 @@ jdk:
 
 android:
   components:
-    - build-tools-21.1.2
-    - android-21
+    - build-tools-23.0.1
+    - android-23
+
+    # Update Android SDK Tools
+    - tools
```

Comment:
I'm surprised this doesn't have to go first, as I thought these were executed in order.